### DISCUSSION TITLE
Why does the wrong animation play when I press the AR button?

### DISCUSSION THREAD

**Discussion by kanthiganesh** [google/model-viewer#1865] [Q&A]
I am trying to animate model in ar mode but only the first animation of the model is playing.
**Browser:Chrome
OS : Android**
this is my code. 
`<model-viewer id="artag" alt="A 3D model of an astronaut." src="RobotExpressive.glb" animation-name="Walking" autoplay ar>`

**Comment by cdata** [comment]
Hi, if you launch AR with the `ar` attribute, you are using Scene Viewer (which is a native Android app). Currently, `<model-viewer>` allows you to configure the playing animation, but Scene Viewer only supports playing the first animation in your model.
